.circleci/config.yml

```diff
@@ -77,10 +77,10 @@ jobs:
       - run:
           name: Dependencies
           command: |
-            ./scripts/wcarthage.sh bootstrap --platform ios --cache-builds --configuration Debug --no-use-binaries
-            ./scripts/wcarthage.sh bootstrap --platform tvos --cache-builds --configuration Debug --no-use-binaries
-            ./scripts/wcarthage.sh bootstrap --platform macos --cache-builds --configuration Debug --no-use-binaries
-            ./scripts/wcarthage.sh bootstrap --platform watchos --cache-builds --configuration Debug --no-use-binaries
+            carthage bootstrap --platform ios --cache-builds --configuration Debug --no-use-binaries --use-xcframeworks
+            carthage bootstrap --platform tvos --cache-builds --configuration Debug --no-use-binaries --use-xcframeworks
+            carthage bootstrap --platform macos --cache-builds --configuration Debug --no-use-binaries --use-xcframeworks
+            carthage bootstrap --platform watchos --cache-builds --configuration Debug --no-use-binaries --use-xcframeworks
```

Comment:
The readme still refers to wcarthage.sh:

https://github.com/mapbox/MapboxGeocoder.swift/blob/ae854d5c27e3e83342200a62fa3587db1ea2bd06/README.md#L44